mslib/msui/mscolab.py

```diff
@@ -653,12 +654,13 @@ def get_recent_pid(self):
             "token": self.token
         }
         r = requests.get(self.mscolab_server_url + '/projects', data=data)
-        _json = json.loads(r.text)
-        projects = _json["projects"]
-        p_id = None
-        if projects:
-            p_id = projects[-1]["p_id"]
-        return p_id
+        if r.text != "False":
```

Comment:
see above.

Reply:
this has to be looked up later. There are more states. The easiest is network connection lost.

But if the session expires there is still a connection, but the user has no access to data. So there these requests don't have json data. 
session expired can be easily done by a server restart and also happen by reaching the expire time on the server. 

A follow up of these changes is #975